open-apparel-registry/open-apparel-registry
# Migrate from django-swagger to drf-yasg

## Overview

Updates the generated Swagger API docs to use `drf-yasg`

Connects #1907 

## Demo

![localhost_8081_api_docs_ (1)](https://user-images.githubusercontent.com/4432106/177417925-73fff2e6-6ff3-4dd4-be7f-3e56afcd3259.png)


## Testing Instructions

* `scripts/update`
* Verify that the output of http://localhost:8081/api/docs/ looks correct

## Checklist
